### ISSUE TITLE
radare2@4.0.0: decompress error

### ISSUE THREAD

**Issue by rpinolf** [ScoopInstaller/Main#551]
```
> scoop update radare2 -f
radare2: 4.0.0 -> 4.0.0
Updating one outdated app:
Updating 'radare2' (4.0.0 -> 4.0.0)
Downloading new version
radare2-msvc_64-4.0.0.zip (4,9 MB) [==========================================================================] 100%
Checking hash of radare2-msvc_64-4.0.0.zip ... ok.
Uninstalling 'radare2' (4.0.0)
Installing 'radare2' (4.0.0) [64bit]
Loading radare2-msvc_64-4.0.0.zip from cache
Extracting radare2-msvc_64-4.0.0.zip ... ERROR Exit code was 2!
Failed to extract files from C:\Users\rpino\scoop\apps\radare2\4.0.0\radare2-msvc_64-4.0.0.zip.
Log file:
  ~\scoop\apps\radare2\4.0.0\7zip.log
```

It seems that current 4.0.0 release is pointing to a corrupted (and small) zip file: **radare2-msvc_64-4.0.0.zip** (4.9 MB)

But it has appeared a new zip with a different naming that seems to be the correct right now:
**radare2-vs2017_64-4.0.0.zip** (48 MB)

Seems this has to be changed.

**Comment by ratatoeey** [comment]
```
> scoop install radare2
Updating Scoop...
Updating 'extras' bucket...
 * 4acbf0d9 mkvtoolnix: Update to version 40.0.0                         10 minutes ago
 * ffb441bf digikam: Update to version 6.4.0                             10 minutes ago
 * 16d4d552 sql-workbench-dev: Update to version 125.3                   70 minutes ago
 * c70931f4 firefox-nightly: Update to version 72.0a1.20191109093718     70 minutes ago
 * 9409f4a8 windowsspyblocker: Update to version 4.25.0                  4 hours ago
 * cf85e4fd googlechrome-dev: Update to version 80.0.3962.2              4 hours ago
 * 2f83cb22 googlechrome-canary: Update to version 80.0.3963.0           4 hours ago
Updating 'main' bucket...
 * e91b6e9d ffmpeg-nightly: Update to version 20191109-bb190de           3 hours ago
 * 1dca8add jx: Update to version 2.0.973                                4 hours ago
Updating 'scoop-completion' bucket...
Scoop was updated successfully!
Installing 'radare2' (4.0.0) [64bit]
Loading radare2-msvc_64-4.0.0.zip from cache
Checking hash of radare2-msvc_64-4.0.0.zip ... ok.
Extracting radare2-msvc_64-4.0.0.zip ... ERROR Exit code was 2!
Failed to extract files from D:\scoop\apps\radare2\4.0.0\radare2-msvc_64-4.0.0.zip.
Log file:
  D:\scoop\apps\radare2\4.0.0\7zip.log

Please try again or create a new issue by using the following link and paste your console output:
https://github.com/ScoopInstaller/Main/issues/new?title=radare2%404.0.0%3a+decompress+error
```

----

regards ratatoeey

**Comment by ratatoeey** [comment]
to fix this.

```
> scoop install radare2
ERROR It looks like a previous installation of radare2 failed.
Run 'scoop uninstall radare2' before retrying the install.
WARN  'radare2' (4.0.0) is already installed.
Use 'scoop update radare2' to install a new version.
> scoop uninstall radare2
Uninstalling 'radare2' (4.0.0).
'radare2' was uninstalled.
> scoop install radare2
Installing 'radare2' (4.0.0) [64bit]
radare2-msvc_64-4.0.0.zip (4.9 MB) [==========================================================================] 100%
Checking hash of radare2-msvc_64-4.0.0.zip ... ok.
Extracting radare2-msvc_64-4.0.0.zip ... ERROR Exit code was 2!
Failed to extract files from D:\scoop\apps\radare2\4.0.0\radare2-msvc_64-4.0.0.zip.
Log file:
  D:\scoop\apps\radare2\4.0.0\7zip.log
```

```
> gc D:\scoop\apps\radare2\4.0.0\7zip.log

7-Zip 19.00 (x86) : Copyright (c) 1999-2018 Igor Pavlov : 2019-02-21

Scanning the drive for archives:
1 file, 5095424 bytes (4976 KiB)

Extracting archive: D:\scoop\apps\radare2\4.0.0\radare2-msvc_64-4.0.0.zip
--
Path = D:\scoop\apps\radare2\4.0.0\radare2-msvc_64-4.0.0.zip
Type = zip
ERRORS:
Unexpected end of archive
Physical Size = 7734375
Characteristics = Local


Sub items Errors: 1

Archives with Errors: 1

Open Errors: 1

Sub items Errors: 1
```

Well, no more radare2 `4.0.0` for me.

----

regards ratatoeey

**Comment by ratatoeey** [comment]
seems like they haven't released binaries for 4.0.0 yet. https://github.com/radareorg/radare2/releases/tag/4.0.0

on a side note,  they have a typo in changes, it says **anal** in heading , then there's **nanal** ,**RAnalFunction.cc**.
----

regards ratatoeey

**Comment by ratatoeey** [comment]
in the meantime, if you don't require the latest verison, use `> scoop install radare2@3.9.0` to install the previous version with binaries.

----

regards ratatoeey
